src/components/thumbnail/collapsible-document-section.tsx

```diff
@@ -18,14 +19,59 @@ interface IProps {
   scale?: number;
   selectedDocument?: string;
   onSelectDocument?: (document: DocumentModelType) => void;
+  subTab: ISubTabSpec;
+  networkResource: INetworkResourceClassResponse;
+  problem: string;
 }
 
 export const CollapsibleDocumentsSection: React.FC<IProps> = observer(
-  ({userName, classNameStr, stores, tab, scale, selectedDocument, onSelectDocument}) => {
+  ({userName, classNameStr, stores, tab, scale, selectedDocument, onSelectDocument, subTab,
+    networkResource, problem}) => {
   const [isOpen, setIsOpen] = useState(false);
   const handleSectionToggle = () => {
     setIsOpen(!isOpen);
   };
+
+  const documentNames: string[] = [];
+  subTab.sections.forEach(section => {
+    if (section.type === "personal-documents") {
+      // get the personal documents
+      networkResource.teachers?.forEach((teacher) => {
+        if (teacher.personalDocuments) {
+          for (const [, document] of Object.entries(teacher.personalDocuments)) {
```

Comment:
If all you care about is the values, you can use `Object.values()` instead of `Object.entries()` here and below.